api/src/test/java/org/pmiops/workbench/actionaudit/auditors/AuthDomainAuditorTest.java

```diff
@@ -16,67 +14,36 @@
 import org.pmiops.workbench.actionaudit.TargetType;
 import org.pmiops.workbench.actionaudit.targetproperties.AccountTargetProperty;
 import org.pmiops.workbench.actionaudit.targetproperties.values.AccountDisabledStatus;
-import org.pmiops.workbench.db.model.DbUser;
-import org.pmiops.workbench.test.FakeClock;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.config.ConfigurableBeanFactory;
 import org.springframework.boot.test.context.TestConfiguration;
 import org.springframework.boot.test.mock.mockito.MockBean;
-import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Import;
-import org.springframework.context.annotation.Scope;
 import org.springframework.test.context.junit4.SpringRunner;
 
 @RunWith(SpringRunner.class)
 public class AuthDomainAuditorTest {
 
   private static final long USER_ID = 101L;
-  private static final long ADMINISTRATOR_USER_ID = 222L;
-  private static final String ADMINISTRATOR_EMAIL = "admin@aou.biz";
-  private static final Instant INSTANT = Instant.parse("2000-01-01T00:00:00.00Z");
-  private static final String ACTION_ID = "9095d2f9-8db2-46c3-8f8e-4f90a62b457f";
-
-  @Autowired private ActionAuditService mockActionAuditService;
-
-  @Captor private ArgumentCaptor<ActionAuditEvent> eventCaptor;
 
   @Autowired private AuthDomainAuditor authDomainAuditAdapter;
+  @MockBean private ActionAuditService mockActionAuditService;
```

Comment:
Cool. I didn't know about this version. I'd been using `@Autowired` even for the mocks and putting a `@MockBean` on the config class.

Reply:
Yeah, I think I only discovered this while digging into some Spring examples out on the internets. I think it's marginally cleaner this way.

Reply:
Yep, there's less duplication and you don't rely only on the name convention.